gateway.yml

```diff
@@ -6,3 +6,5 @@ resourceConfig: org.radarcns.gateway.inject.EcdsaJwtGatewayResources
 jwtKeystorePath: src/integrationTest/docker/etc/config/keystore.p12
 jwtKeystoreAlias: radarbase-managementportal-ec
 jwtKeystorePassword: radarbase
+jwtECPublicKeys: []
```

Comment:
I have a question. If we are using yaml files, should not the convention to read list is
```
jwtECPublicKeys:
  - Key1
  - Key2
```